README.md

```diff
@@ -14,7 +14,15 @@ Install
 
 * Generate the `hpi` file with the command: `mvn package`
 
+If you don't have Maven installed you can build the plugin using a Docker container.
+
+Just clone this repository and run :
+```
+docker run -it --rm --name logstash-plugin -v "$(pwd)":/usr/src/mymaven -w /usr/src/mymaven maven:3.3-jdk-8 mvn package
```

Comment:
I'm not sure how useful this is, maybe we could just add https://github.com/takari/maven-wrapper that way you only need JDK 8 installed